### DISCUSSION TITLE
Go to CWD

### DISCUSSION THREAD

**Discussion by TroySigX** [nvim-tree/nvim-tree.lua#1852] [Q&A] [answered]
In NerdTree, when I type CD, the root will immediately switch to the current working dir. How do i do that in Nvim-Tree?

**Comment by alex-courtis** [comment, marked as answer]
see `:help nvim-tree.sync_root_with_cwd`

**Reply by TroySigX** [reply]
Thanks, but that wasn't what I meant. I wanted to know, for example, when I go up to parent directories multiple times, and want to go back to my current working dir (which is done by typing CD, upper-case, in nerdtree), how do I do it?

**Reply by alex-courtis** [reply]
Added new recipe: https://github.com/nvim-tree/nvim-tree.lua/wiki/Recipes#change-root-to-global-current-working-directory

**Reply by TroySigX** [reply]
Can you tell me how to move the cursor to the root's current directory?

**Reply by TroySigX** [reply]
Thanks! That helps a lot.

**Reply by alex-courtis** [reply]
Luaish answer:

```lua
vim.cmd.normal({ args = { "gg" }, bang = true })
```

**Reply by alex-courtis** [reply]
Awesome. Please mark as answer :)
